README.rst

```diff
@@ -261,34 +261,34 @@ The following bundles are available:
 Serializers
 ~~~~~~~~~~~
 
-:``celery[auth]``:
+: ``celery[auth]``:
```

Comment:
Could you please explain the new format? It does not display so well in the [preview](https://github.com/celery/celery/pull/5787/files?short_path=88b99bb#diff-88b99bb28683bd5b7e3a204826ead112), but I am not sure it is rendered 100% accurately.